scripts/booster-analysis.sh

```diff
@@ -48,7 +48,7 @@ run_tarball(){
 export -f run_tarball
 export SCRIPT_DIR
 
-find $BUG_REPORT_DIR -name \*.tar -or -name \*.tar.gz -print0 | xargs -0 -t -I {} -P $PARALLEL bash -c 'run_tarball "$@"' $(basename {}) {}
+find $BUG_REPORT_DIR -name \*.tar -print0 -or -name \*.tar.gz -print0 | xargs -0 -t -I {} -P $PARALLEL bash -c 'run_tarball "$@"' $(basename {}) {}
```

Comment:
This still doesn't somehow work for me, see here:

```
find scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint -print0 -name \*.tar -or -name \*.tar.gz | xargs -0 du -sh
3.4G	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint
8.0K	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/.DS_Store
3.4G	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove
 12K	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/.DS_Store
704M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol
 35M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-countertest-testincrement-0-spec.tar
 31M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-expectreverttest-test_expectrevert_message-0-spec.tar
 29M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-emitcontracttest-testexpectemit-0-spec.tar
 29M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-allowchangestest-testallow-0-spec.tar
 30M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-arithmetictest-test_wdiv_rounding-uint256-uint256-0-spec.tar
 29M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-expectcalltest-testexpectregularcall-0-spec.tar
 33M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-owneruponlytest-testincrementasowner-0-spec.tar
 29M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-storetest-teststoreload-0-spec.tar
 32M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-arithmetictest-test_wmul_weakly_increasing_positive-uint256-uint256-0-spec.tar
 29M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-emitcontracttest-testexpectemitdonotcheckdata-0-spec.tar
 29M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-expectcalltest-testexpectstaticcall-0-spec.tar
 29M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-allowchangestest-testfailallowcallstoaddress-0-spec.tar
 29M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-emitcontracttest-testexpectemitcheckemitter-0-spec.tar
 29M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-safetest-testwithdrawfuzz-uint96-0-spec.tar
 35M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-expectreverttest-testfail_expectrevert_false-0-spec.tar
 35M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-expectreverttest-test_expectrevert_bytes4-0-spec.tar
 32M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-allowchangestest-testfailallowchangestostorage-0-spec.tar
 29M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-allowchangestest-testallow_fail-0-spec.tar
 29M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-arithmetictest-test_max1_broken-uint256-uint256-0-spec.tar
 31M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-expectreverttest-testfail_expectrevert_bytes4-0-spec.tar
 31M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-expectreverttest-testfail_expectrevert_failandsuccess-0-spec.tar
 29M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-storetest-testaccesses-0-spec.tar
 31M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-expectreverttest-test_expectrevert_returnvalue-0-spec.tar
1.0G	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/erc20
 12K	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/erc20/.DS_Store
...
```

vs

```
find scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint -name \*.tar -print0 | xargs -0 du -sh
 35M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-countertest-testincrement-0-spec.tar
 31M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-expectreverttest-test_expectrevert_message-0-spec.tar
 29M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-emitcontracttest-testexpectemit-0-spec.tar
 29M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-allowchangestest-testallow-0-spec.tar
 30M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-arithmetictest-test_wdiv_rounding-uint256-uint256-0-spec.tar
 29M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-expectcalltest-testexpectregularcall-0-spec.tar
 33M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-owneruponlytest-testincrementasowner-0-spec.tar
 29M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-storetest-teststoreload-0-spec.tar
 32M	scripts/bug-reports/kevm-v1.0.519-sam-implies-endpoint/test_pyk_prove/kontrol/test-arithmetictest-test_wmul_weakly_increasing_positive-uint256-uint256-0-spec.tar
 ...
```

i.e. the first command is listing folders, not just tar files

Reply:
Ah, nm, i copied the command wrong